build/js/bootstrap-tour.js

```diff
@@ -453,16 +457,32 @@
       }
     };
 
-    Tour.prototype._isRedirect = function(path, currentPath) {
+    Tour.prototype._isRedirect = function(host, path, currentPath) {
+      var current_host;
+      if (host !== '') {
+        current_host = document.location.href.substr(0, document.location.href.lastIndexOf(document.location.pathname));
```

Comment:
`host` is use to redirect the step to a specific location.  `window.location.hostname` and `window.location.host` don't return the complete host with protocole.